src/pages/baan-profile.tsx

```diff
@@ -0,0 +1,235 @@
+import Image from 'next/image';
+
+const profilePic = '/images/pfp-placeholder.svg';
+
+const BaanProfile = () => {
+    const bgStyle: string = 'lg:mb-none lg:mb-6 p-auto py-8 px-8';
```

Comment:
If you want to add a new custom style group, you can do so in `src/styles/globals.css`, like in the example below.
```css
@layer components {
    // ... other components go here
    .bg-style {
        @apply lg:mb-none lg:mb-6 p-auto py-8 px-8;
    }
}
```